src/helpers/Box.cpp

```diff
@@ -109,29 +109,41 @@ CBox& CBox::expand(const double& value) {
     return *this;
 }
 
+CBox& CBox::shrink(const double& value) {
```

Comment:
expand takes negative values doesn't it?

Reply:
fixed